lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -56,14 +56,14 @@ protected function getNextPageUri($data, $parameters, $endpoint) {
   /**
    * Get the URI for the previous page in the collection of data.
    *
-   * @param array $data
+   * @param  array $data
    *   An associative array of pagination parameters:
    *   - total: (int) Complete total number of available data items found.
    *   - per_page: (int) Total number of data items to show per page.
    *   - current_page: (int) Current page number.
    *   - total_pages: (int) Total number of pages available.
-   * @param array $parameters An associative array of current location parameters.
-   * @param string $endpoint Endpoint for building URI.
+   * @param  array $parameters An associative array of current location parameters.
```

Comment:
No lining up of the description, just the data type and the variable name. The variable names can range in size significantly, so that'll just be a nightmare :scream: